src/Collisions/babylon.collisionCoordinator.ts

```diff
@@ -390,7 +392,8 @@ module BABYLON {
             // Check all meshes
             for (var index = 0; index < this._scene.meshes.length; index++) {
                 var mesh = this._scene.meshes[index];
-                if (mesh.isEnabled() && mesh.checkCollisions && mesh.subMeshes && mesh !== excludedMesh) {
+                if(!mesh._collisionGroup) mesh._collisionGroup = -1;

```

Comment:
i'm assuming the excludedMesh is null when it is a freeCamera moving, so it's mask would be overridden to -1 and collide with everything